packages/utilities/src/Customizer.tsx

```diff
@@ -1,171 +0,0 @@
-import * as React from 'react';
```

Comment:
Deleting this file could be a breaking change if anyone was importing it directly. Could you leave a placeholder?

Reply:
Nobody should be using a filepath import on utilities. We only officially support filepath imports for OUFR top level files. Internal filenames, their path, their casing, should not be a contract.